platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/event/GossipEvent.java

```diff
@@ -243,7 +250,7 @@ public long getGeneration() {
      * @return the birth round of the event
      */
     public long getBirthRound() {
```

Comment:
I agree with @alittley regaridng the naming. Now we have a `setBirthRoundOverride` but a `getBirthRound`. That is not really consinstent.

Reply:
I am tagging @cody-littley since I think he originally wrote the code so he can comment.

But I think `setBirthRound` is more misleading. `birthRound` is an immutable part of an event, it cannot be set after an event is created. So even if the override is changed, the birth round remains the same. The override seems to be some kind of migration trick (don't know the exact details) that will be used only once when migrating and will be deleted afterwards.

Reply:
Bearing this in mind, it seems that the member should be called `birthRound`, and the method should be called `overrideBirthRound`

Reply:
I'm a fan of the naming schema

- `getBirthRound()`
- `overrideBirthRound(long birth round)`

This getter method is intentionally named like a regular getter. After the birth round migration, this method will actually be just a regular getter (since the override is a migration-only construct). All parts of the code should use this getter the same way they would if there was no such thing as an overriding birth round.

Reply:
renamed